core/tests/karma.conf.ts

```diff
@@ -176,7 +176,7 @@ module.exports = function(config) {
             loader: 'underscore-template-loader'
           },
           {
-            test: /\.ts$/,
+            test: /^(?!.*(s|S)pec\.ts$).*\.ts$/,
```

Comment:
Maybe add a comment here to explain the rationale.

Reply:
Done